src/OpenFeature/Api.cs

```diff
@@ -38,25 +39,48 @@ public sealed class Api : IEventBus
         private Api() { }
 
         /// <summary>
-        /// Sets the feature provider. In order to wait for the provider to be set, and initialization to complete,
+        /// Sets the default feature provider to given clientName without awaiting its initialization.
+        /// </summary>
+        /// <remarks>The provider cannot be set to null. Attempting to set the provider to null has no effect.</remarks>
+        /// <param name="featureProvider">Implementation of <see cref="FeatureProvider"/></param>
+        [Obsolete("Will be removed in later versions; use SetProviderAsync, which can be awaited")]
+        public void SetProvider(FeatureProvider featureProvider)
+        {
+            this.EventExecutor.RegisterDefaultFeatureProvider(featureProvider);
+            this._repository.SetProvider(featureProvider, this.GetContext()).ConfigureAwait(false);
+        }

```

Comment:
Nice catch. Yes, this was just copy/paste - the PR was more of a "thinking out loud" experiment to resolve our release conundrum.

Reply:
I've removed the `.ConfigureAwait(false);` and added a discard to avoid [CS4014](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs4014).